src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -320,6 +331,7 @@ def def_meta_data():
         # "timeIncrement": 'timeIncrement',
         "earthRadiusCurvature": 'earthLocalRadiusOfCurvature',
         "satelliteIdentifier": 'satelliteIdentifier',
+        "satelliteSubIdentifier": 'satelliteSubIdentifier',
```

Comment:
the addition of this line will make this converter **not** backward compatible
will need to add handling at line:
https://github.com/JCSDA-internal/ioda-converters/blob/40531cd029207f88a7c015b78736e5904079bb66/src/gnssro/gnssro_bufr2ioda.py#L160

Reply:
have a way to handle this now